xivo_fetchfw/package.py

```diff
@@ -678,9 +681,10 @@ def __init__(self, installable_pkg_sto, installed_pkg_sto,
         self._ignore = [] if ignore is None else ignore
         self._nodeps = nodeps
 
-    def _upgrade_list_filter_function(self, (installed_pkg, installable_pkg)):
+    def _upgrade_list_filter_function(self, xxx_todo_changeme):
         # Return true if installed_pkg is not in the ignore list and if
         # installable_pkg version is higher than installed_pkg version
+        (installed_pkg, installable_pkg) = xxx_todo_changeme
```

Comment:
Wanna change this variable name? Something like `xxx_todo_changeme_please_pretty_please_i_m_ugly_and_i_dont_want_to_exist_please_press_that_delete_button_111` maybe? :D

Reply:
Ahhhh that tool kills me everytime with variables like these :joy: